src/App.css

```diff
@@ -1,42 +1,295 @@
+  border-color: #007bff;
+}
+
+.searchbutton {
+  padding: 10px;
+  border: none;
+  background-color: #007bff;
+  color: white;
+  cursor: pointer;
+  border-radius: 0 5px 5px 0;
+  font-size: 16px;
+  transition: background-color 0.3s ease;
+}
+
+.searchbutton:hover {
+  background-color: #0056b3;
+}
+
+/* Responsive adjustments */
+@media (max-width: 90%) {
+  .header-centre {
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
   }
-  to {
-    transform: rotate(360deg);
+
+  .navigator {
+    display: none;
+  }
+
+  .header {
+    padding: 10px;
+  }
+
+  .searchbar {
+    width: 100%;
+  }
+
+  .searchbar input[type="text"] {
+    width: 75%;
   }
-}
 
-@media (prefers-reduced-motion: no-preference) {
-  a:nth-of-type(2) .logo {
-    animation: logo-spin infinite 20s linear;
+  .searchbutton {
+    font-size: 14px;
   }
+
+  .nav-list {
+    display: none;
+  }
+}
+
+/* Item card styling */
+.item-card {
+  position: relative;
+  overflow: hidden;
+  cursor: pointer;
+  transition: transform 0.3s;
+  text-align: center;
+}
+
+.item-card:hover {
+  transform: scale(1.05);
+}
+
+.item-card .hover-info {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: rgba(0, 0, 0, 0.7);
+  color: white;
+  padding: 10px;
+  border-radius: 5px;
+  display: none;
+  text-align: center;
+  z-index: 1;
+}
+
+.item-card:hover .hover-info {
+  display: block;
+}
+
+.solo-image {
+  border: solid black 5px;
+  border-radius: 15%;
+  padding: 10px;
+  height: 700px;
+  width: 700px;
+}
+
+.title {
+  text-align: center;
+  font-size: 50px;
+  padding: 1px;
+}
+
+.description-div {
+  height: 700px;
+  width: 1000px;
+}
+
+.description {
+  line-height: 1.6;
+  font-size: 20px;
+}
+
+.button-container {
+  display: flex;
+  justify-content: center;
+  padding-top: 30px;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 20px;
+  padding: 10px;
+  align-items: center;
+  margin: 100px;
+}
+
+.image {
+  height: 300px;
+  border: solid black 5px;
+  border-radius: 15%;
+  padding: 10px;
+}
+
+.basket-count {
+  display: grid;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  font-size: 2.5rem;
+  padding: 0;
+  text-decoration: none;
+}
+
+.img-container {
+  position: relative;
+  text-align: center;
+  color: white;
+}
+
+.top-left {
+  font-size: medium;
+  color: pink;
+  position: absolute;
+  bottom: 20px;
+  text-align: center;
+  position: relative;
+}
+
+.comments-section {
+  margin: 20px;
+  padding: 10px;
+}
+
+.comment-card {
+  background-color: #f9f9f9;
+  border: 1px solid #ddd;
+  border-radius: 8px;
+  padding: 15px;
+  margin-bottom: 10px;
+  box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.1);
+}
+
+.comment-card p {
+  margin: 8px 0;
+  font-size: 14px;
 }
 
-.card {
-  padding: 2em;
+.comment-card b {
+  font-size: 16px;
 }
 
-.read-the-docs {
-  color: #888;
+.comment-card:hover {
+  background-color: #f0f0f0;
```

Comment:
So, I'm using a dark theme... You may need to remove a lot of the default css as this is how comments render for me.
![image](https://github.com/user-attachments/assets/bf1dce5d-f3f4-454a-a127-4ee95f793dcb)